### PR TITLE
e2e: device plugin gets deleted before next one is deployed

### PR DESCRIPTION
**Pull request by hj-johannes-lee**
fixes: #1364
Signed-off-by: Hyeongju Johannes Lee <hyeongju.lee@intel.com>

### CODE REVIEW

File: test/e2e/qat/qatplugin_dpdk.go [hunk 1]
```diff
@@ -90,6 +93,14 @@ func describeQatDpdkPlugin() {
 			}
 		})
 
+		ginkgo.AfterEach(func() {
+			ginkgo.By("undeploying DSA plugin")
```
Comment (mythi) [inline test/e2e/qat/qatplugin_dpdk.go:97]:
```suggestion
			ginkgo.By("undeploying QAT plugin")
```

File: test/e2e/qat/qatplugin_kernel.go [hunk 2]
```diff
@@ -76,6 +80,14 @@ func describeQatKernelPlugin() {
 			}
 		})
 
+		ginkgo.AfterEach(func() {
+			ginkgo.By("undeploying DSA plugin")
```
Comment (mythi) [inline test/e2e/qat/qatplugin_kernel.go:84]:
```suggestion
			ginkgo.By("undeploying QAT plugin")
```

File: test/e2e/qat/qatplugin_dpdk.go (hunk 1 above)
Comment (hj-johannes-lee) [reply test/e2e/qat/qatplugin_dpdk.go:97]: Ah, thanks. I chnaged others, but forgot for QAT. Let me fix them.

File: test/e2e/qat/qatplugin_kernel.go [hunk 3]
```diff
@@ -76,6 +80,14 @@ func describeQatKernelPlugin() {
 			}
 		})
 
+		ginkgo.AfterEach(func() {
+			ginkgo.By("undeploying QAT plugin")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "delete", "-k", filepath.Dir(yamlPath))
```
[the remark this answers is not in the document]
Comment (hj-johannes-lee) [reply test/e2e/qat/qatplugin_kernel.go]: fixed!

**Review by mythi — APPROVED** [review]

**Pull request merged**